packages/nodes-base/nodes/Merge/test/v3/utils.test.ts

```diff
@@ -0,0 +1,92 @@
+import type { IDataObject, INodeExecutionData, IPairedItemData } from 'n8n-workflow';
+
+import { modifySelectQuery, rowToExecutionData } from '../../v3/helpers/utils';
+
+describe('rowToExecutionData', () => {
+	test('should return empty json and pairedItem when input is empty', () => {
```

Comment:
To build up on previous comment.
```suggestion
	test('returns empty json and pairedItem when input is empty', () => {
```